pkg/operator/ceph/spec/spec.go

```diff
@@ -102,7 +102,7 @@ func PodVolumes(dataDirHostPath, namespace string, confGeneratedInPod bool) []v1
 	return []v1.Volume{
 		{Name: k8sutil.DataDirVolume, VolumeSource: dataDirSource},
 		configVolume,
-		StoredLogVolume(path.Join(dataDirHostPath, "log", namespace)),
+		StoredLogVolume(path.Join(dataDirHostPath, namespace, "log")),
```

Comment:
I have it here too: https://github.com/rook/rook/pull/3316/commits/aa8e85386ce08f979ad0fa1f87ed06722070eb90

Reply:
What PR is that? Is it in master yet?

Reply:
No no, this will likely go first and I'll rebase no worries :). I was just pointing it out.